protocol/packages/currencies/src/lease/mod.rs

```diff
@@ -6,14 +6,14 @@ use sdk::schemars::{self, JsonSchema};
 use crate::PaymentGroup;
 
 #[cfg(not(feature = "testing"))]
-use self::r#impl as impl_mod;
+pub(crate) use self::r#impl as impl_mod;
 #[cfg(feature = "testing")]
-use self::testing as impl_mod;
+pub(crate) use self::testing as impl_mod;
 
 #[cfg(not(feature = "testing"))]
-mod r#impl;
+pub(crate) mod r#impl;
 #[cfg(feature = "testing")]
-pub mod testing;
+pub(crate) mod testing;
```

Comment:
Just for clarity, what required this change? I understand the reduction of visibility, but what made extending the visibility necessary?

Reply:
the need to define Stable to be one of the lease currencies

Reply:
so it's harmless to show all payment currencies inside the crate for use as Stable in short protocols